src/orga/governance.rst

```diff
@@ -31,6 +31,11 @@ Here are defined the primary teams participating in conda-forge activities.
   core, part of staged-recipes, or maintainers. This includes first time
   contributors, collaborators, and funders. They have no special rights within
   the conda-forge organization itself.
+* **emeritus-core:** Core members that are inactive (commits, GitHub comments/issues/reviews,
+  and voting on polls) in the past six months will be asked if they want to become emeritus-core
```

Comment:
It would be good to have a clause to people to self select as emeritus (before 6 months), creating a sabbatical like situation. We also might need a way to come off the emeritus status, and something that records the current status of the core team.

Reply:
> It would be good to have a clause to people to self select as emeritus (before 6 months), creating a sabbatical like situation.

I don't think we need that in there b/c I guess anyone can ask for a core member to be emeritus, even the person becoming emeritus :smile: 

> We also might need a way to come off the emeritus status, and something that records the current status of the core team.

Indeed that is not clear. Implicitly we agreed that all someone has to do was to ask to be back. We should write that in there.

Reply:
Does it make sense to include the dev meetings in the list of things that is called out here?

Reply:
I think so. I'll amend it tomorrow.